Netkan/Model/Metadata.cs

```diff
@@ -5,20 +5,22 @@ namespace CKAN.NetKAN.Model
 {
     internal sealed class Metadata
     {
-        private const string KrefPropertyName = "$kref";
-        private const string VrefPropertyName = "$vref";
+        private const string KrefPropertyName        = "$kref";
+        private const string VrefPropertyName        = "$vref";
         private const string SpecVersionPropertyName = "spec_version";
-        private const string VersionPropertyName = "version";
-        private const string DownloadPropertyName = "download";
+        private const string VersionPropertyName     = "version";
+        private const string DownloadPropertyName    = "download";
+        public  const string UpdatedPropertyName     = "x_netkan_asset_updated";
```

Comment:
How does this field in the Netkan file get set?

Reply:
That's added to GithubTransformer.cs, when it's translating a GitHub release into a Metadata object:

https://github.com/KSP-CKAN/CKAN/pull/2337/files#diff-2de26526c75bb7d8973d43c0a2581a4bR79

The "x_netkan" prefix is to make sure it gets stripped out before we make the .ckan file (which we would change if we decide we want this data to persist). And we have the option to populate this field in other transformers if we want this for SpaceDock in the future, for example.